### ISSUE TITLE
y_true_class和pred_class 不一致的问题

### ISSUE THREAD

**Issue by gaofssvm** [zzh8829/yolov3-tf2#262]
在y_true_class,是没有进行onehot编码的，两个维数不一致，这样对计算结果有影响吗，并且需要对onehot编码进行smooth处理

**Comment by w19787** [comment]
this project doesn't convert the y_true to onehot. that is the reason the sparse cross entropy function is used for class loss calculation. You can convert to one hot and apply label smoothing but you need to check relative functions to be consistent, including loss function.

**Comment by Sarahu1** [comment]
您好，请教一下，咱们这个算法默认识别88种类，如果我只想识别车和人怎么修改？多谢
